strawberry/fastapi/handlers/graphql_ws_handler.py

```diff
@@ -0,0 +1,62 @@
+from contextlib import suppress
+from typing import Any, Optional
+
+from starlette.websockets import WebSocket, WebSocketDisconnect, WebSocketState
+
+from strawberry.schema import BaseSchema
+from strawberry.subscriptions import GRAPHQL_WS_PROTOCOL
+from strawberry.subscriptions.protocols.graphql_ws.handlers import BaseGraphQLWSHandler
+from strawberry.subscriptions.protocols.graphql_ws.types import OperationMessage
+
+
+class GraphQLWSHandler(BaseGraphQLWSHandler):
+    def __init__(
+        self,
+        schema: BaseSchema,
+        debug: bool,
+        keep_alive: bool,
+        keep_alive_interval: float,
+        context,
+        root_value,
```

Comment:
Is it common for those to not have type hint?

Reply:
Yes